xe-daemon/xe-daemon.go

```diff
@@ -125,6 +128,14 @@ func main() {
 					}
 				}
 			}
+			if count%DivisorMax == 0 {
```

Comment:
This logic can be called after "if updated"
So we will not have a "if count%DivisorMax == 0" Check.